python/py-gcn/Portfile

```diff
@@ -6,10 +6,9 @@ PortGroup           github 1.0
 
 github.setup        lpsinger pygcn 0.1.19 v
 name                py-gcn
-python.versions     27 35 36 37
+python.versions     37
```

Comment:
```suggestion
python.versions     37 38 39 310
```

Reply:
while I in principle agree with adding subports, I will not do so in this PR which is purely about removing EOL versions. Adding newer Python subports requires one to make sure upstream supports them and such, that's something for a maintainer to do at another time.